### ISSUE TITLE
[Question/Feature] Define custom libvirt storage pool location in install-config.yml

### ISSUE THREAD

**Issue by oswee** [openshift/installer#2535]
Is there an documentation or option to define custom storage pool location for libvirt installation?
Would like to have something like:
```yaml
...
platform:
  libvirt:
    URI: qemu+tcp://192.168.122.1/system
    network:
      if: tt0
    pool:
      path: /my/custom/pool/path
      name: openshift
...
```
Currently installer uses default libvirt pool location in `/var/lib/libvirt/images` which in my case is specific drive with some constraints and i can't use that.
I have `default` pool defined as well (in custom location), but seems openshift-installer ignores that.
So second option (less preferable) would be to check, is `default` pool defined and if so, then to place ocp volumes there.

How can i force openshift-installer to use custom pool location? Should i modify Go code before compiling installer?

**Comment by oswee** [comment]
Ok. I think i solved it simply by:

```sh
$ mkdir /home/user/kvm/pools/openshift-images

# semanage fcontext -t virt_image_t -a '/home/user/kvm/pools/(/.*)?'
# restorecon /home/user/kvm/pools

# rmdir /var/lib/libvirt/openshift-images/
# ln -s /home/user/kvm/pools/openshift-images/ /var/lib/libvirt/openshift-images
```

**Comment by zeenix** [comment]
Yes, symlinking is the usual solution. PR to customize the pool location welcome though, in which case please reopen this issue.

/close

**Comment by marshallford** [comment]
I followed the deleted user's symlink advice and I'm running into this Terraform error on bootstrap:

```
...
ERROR
ERROR Error: Error creating libvirt domain: virError(Code=1, Domain=10, Message='internal error: qemu unexpectedly closed the monitor: 2020-10-26T20:53:30.795245Z qemu-kvm: -fw_cfg name=opt/com.coreos/config,file=/var/lib/libvirt/openshift-images/fog-rbw9q/fog-rbw9q-master.ign: can't load /var/lib/libvirt/openshift-images/fog-rbw9q/fog-rbw9q-master.ign')
ERROR
ERROR   on ../tmp/openshift-install-157045713/main.tf line 76, in resource "libvirt_domain" "master":
ERROR   76: resource "libvirt_domain" "master" {
ERROR
ERROR
ERROR
...
```
